pkg/resource/events-logger-secret/reconciler.go

```diff
@@ -0,0 +1,120 @@
+package eventsloggersecret
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	apimachineryerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/giantswarm/logging-operator/pkg/common"
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+	loggingcredentials "github.com/giantswarm/logging-operator/pkg/resource/logging-credentials"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// Reconciler implements a reconciler.Interface to handle
+// Events-logger secret: extra events-logger secret about where and how to send logs (in this case : k8S events)
+type Reconciler struct {
+	client.Client
+}
+
+// ReconcileCreate ensures events-logger-secret is created with the right credentials
+func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Interface) (ctrl.Result, error) {
+	logger := log.FromContext(ctx)
+	logger.Info("events-logger-secret create")
+
+	// Retrieve secret containing credentials
+	var eventsLoggerCredentialsSecret v1.Secret
+	err := r.Client.Get(ctx, types.NamespacedName{Name: loggingcredentials.LoggingCredentialsSecretMeta(lc).Name, Namespace: loggingcredentials.LoggingCredentialsSecretMeta(lc).Namespace},
```

Comment:
Maybe extract  loggingcredentials.LoggingCredentialsSecretMeta(lc) into a variable?

Reply:
Oh you just took that